### PR TITLE
chore: fix clippy lint `comparison_chain`

### PR DESCRIPTION
**Pull request by theHamsta**

### CODE REVIEW

File: src/project.rs [hunk 1]
```diff
@@ -378,12 +379,16 @@ impl Projects {
...
 
         // strange solution to make gtk assertions happy
-        if previous_height < treeview_height {
-            self.scroll.set_max_content_height(treeview_height);
-            self.scroll.set_min_content_height(treeview_height);
-        } else if previous_height > treeview_height {
-            self.scroll.set_min_content_height(treeview_height);
-            self.scroll.set_max_content_height(treeview_height);
+        match previous_height.cmp(&treeview_height) {
+            Less => {
```
Comment (theHamsta) [inline src/project.rs:383]: I'm not sure whether this improves readability. It at least forces to consider all there cases

Comment (Lyude) [reply src/project.rs:383]: TBH I think I'd prefer these ones be `#[clippy:allow(…)]`d

Comment (Lyude) [reply src/project.rs:383]: TBH I think I'd prefer this ones be `#[clippy:allow(…)]`d

**Review by Lyude — CHANGES_REQUESTED** [review]

File: src/tabline.rs [hunk 2]
```diff
@@ -109,45 +110,49 @@ impl Tabline {
...
-                let state_ref = Rc::clone(&self.state);
-                close_btn.connect_clicked(move |btn| {
-                    let current_label = btn.parent().unwrap();
-                    for i in 0..tabs.n_pages() {
-                        let page = tabs.nth_page(Some(i)).unwrap();
-                        let label = tabs.tab_label(&page).unwrap();
-                        if label == current_label {
-                            state_ref.borrow().close_tab(i);
+        match count.cmp(&tabs.len()) {
+            Less => {
```
Comment (theHamsta) [inline src/tabline.rs:114]: I guess if one would like to use `cmp` one would need to factor out the cases to separate functions.

**Comment by Lyude** [comment]
JFYI: I was fine with pretty much all the other lints here, it was just the one hunk I commented on :P, you're welcome to re-open if you want

**Comment by theHamsta** [comment]
> JFYI: I was fine with pretty much all the other lints here, it was just the one hunk I commented on :P, you're welcome to re-open if you want

Ah, ok. I thought you were against this lint alltogether

**Pull request closed without merging**